docker-images/kafka-based/kafka/cruise-control-scripts/cruise_control_tls_prepare_certificates.sh

```diff
@@ -2,25 +2,10 @@
 set -e
 set +x
 
-# Parameters:
-# $1: Path to the new truststore
-# $2: Truststore password
-# $3: Public key to be imported
-# $4: Alias of the certificate
-function create_truststore {
-   keytool -keystore "$1" -storepass "$2" -noprompt -alias "$4" -import -file "$3" -storetype PKCS12
-}
-
-# Parameters:
-# $1: Path to the new keystore
-# $2: Truststore password
-# $3: Public key to be imported
-# $4: Private key to be imported
-# $5: CA public key to be imported
-# $6: Alias of the certificate
-function create_keystore {
-   RANDFILE=/tmp/.rnd openssl pkcs12 -export -in "$3" -inkey "$4" -name "$6" -password pass:"$2" -out "$1"
-}
+# Load predefined functions for preparing trust- and keystores
+# Shellcheck SC1091 is disabled because the path does not work locally, but does inside the container
+# shellcheck disable=SC1091
+source ../kafka/tls_utils.sh
```

Comment:
```suggestion
source ../scripts/tls_utils.sh
```

shouldn't be the path be like that?

Reply:
No, that is not the location in the container images.